common/test/acceptance/tests/studio/test_studio_library.py

```diff
@@ -343,6 +343,65 @@ def test_delete_shifts_blocks(self):
         self.assertEqual(self.lib_page.xblocks[-1].name, '11')
         self.assertEqual(self.lib_page.get_page_number(), '1')
 
+    def test_previews(self):
+        """
+        Scenario: Ensure the user is able to hide previews of XBlocks.
```

Comment:
These would read easier if you used "And" and "Then". For instance,

Given that I have a library in Studio with 40 XBlocks
Then previews are visible
And when I click the toggle previews button
Then the previews will not be visible
And when I click the toggle previews button
Then the previews are visible